app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv_command, first_file, second_file_with_path = command.split()
```

Comment:
This line must be after ` len(command.split()) == 3:` because if len is less than 3, the code will be broken here.